### ISSUE TITLE
Load the 3D scene cost too much time, and the browser will load the same scene as the before I loaded, is there a good solution? Thank you very much.

### ISSUE THREAD

**Issue by RongkunYang** [francisengelmann/PyViz3D#8]

**Comment by yrk-creator** [comment]
BrokenPipeError: [Errno 32] Broken pipe

This is the common error, why did this error occurs?

**Comment by francisengelmann** [comment]
Hi @yrk-creator,
Thanks for the feedback! Would it be possible to share a minimal example code that causes this error?

**Comment by yrk-creator** [comment]
Hi, thank you for your fast response. 
I am using this tools to visualize the 3D scene in ScanNet and S3DIS, the visualization code is the same as it is in "Mask3D". When I visualize the scene, it seems to load for a long time, and sometimes it suggests the "BrokenPipeError: [Errno 32] Broken pipe".

**Comment by yrk-creator** [comment]
Is there any requirements for browsers, or some settings should be opened?  Thank you.

**Comment by francisengelmann** [comment]
If you are using Chrome, could you please share the output of 'View/Developer/Javascript Console'?

**Comment by yrk-creator** [comment]
![image](https://user-images.githubusercontent.com/71752017/223131428-55d91e7c-7891-46e8-8b95-8b3920da305a.png)

**Comment by yrk-creator** [comment]
This is the point cloud scene, but the cosole outputs nothing.

**Comment by yrk-creator** [comment]
I will check the point cloud data again, to ensure where the problem lies in, thank you.

**Comment by yrk-creator** [comment]
![image](https://user-images.githubusercontent.com/71752017/223288492-c83549d5-5f2c-4482-8d6b-f1270414811e.png)

**Comment by yrk-creator** [comment]
I ensured that there is no errors in the point cloud data, so the problem lies in the browser. When I load the scene, the Instance%20(GT) that doesn't rendered has no Request head, but the loaded points get the request head.

**Comment by yrk-creator** [comment]
![image](https://user-images.githubusercontent.com/71752017/223288993-69940cae-cc43-4b82-ae53-cb2e444f7d0d.png)

**Comment by yrk-creator** [comment]
This is the console output, hope this imformation could be useful, thank you.

**Comment by francisengelmann** [comment]
Thanks, I see you are using Mask3D, I will try to replicate the error here.

**Comment by yrk-creator** [comment]
OK, thank you, the scene shows normally after serveral hours, but it takes a long time. So if there are some settings with the browsers would be better.

**Comment by SARIHUST** [comment]
> ![image](https://user-images.githubusercontent.com/71752017/223131428-55d91e7c-7891-46e8-8b95-8b3920da305a.png)

Hi @RongkunYang, @francisengelmann 
I've encountered the same problems, and after clearing Chrome's cache I got the right results. However, clearing the cache each time I visualize a scene is quite frustrating, so I want to know if you have any better solutions.

**Comment by SARIHUST** [comment]
Thanks for the reply @francisengelmann. 
Unfortunately, if I use these keyboard shortcuts to reload the page, I will directly run into the faulty visualization problem mentioned above. And if I simply use F5 to reload, the browser gets the data from the disk cache, leading to visualizing the previous scene.
I found a better solution for my case today. I used F12 to open the developer tools and selected 'disable cache'. Then when I keep the developer tools page open and reload the page with F5, I can get the new scene data.
